lib/jira/client/index.js

```diff
@@ -62,57 +57,26 @@ module.exports = async (id, installationId, jiraHost) => {
         return text.match(jiraIssueRegex)
       },
       comments: {
-        // eslint-disable-next-line camelcase
-        getForIssue: (issue_id) => instance.get(`/rest/api/latest/issue/:issue_id/comment`, {
-          fields: {
-            issue_id
-          }
-        }),
-        // eslint-disable-next-line camelcase
-        addForIssue: (issue_id, payload) => instance.post(`/rest/api/latest/issue/:issue_id/comment`, payload, {
-          fields: {
-            issue_id
-          }
-        })
+        getForIssue: (issueId) => instance.get(`/rest/api/latest/issue/${issueId}/comment`),
```

Comment:
Is there any implied risk of exploitation or anything else when replacing the placeholder + arguments syntax with string interpolation?